go.md

```diff
@@ -0,0 +1,24 @@
+# Working with Go language versions in project
+
+Upon the release of a new version of Go, all repositories are updated
```

Comment:
And additional section like "Version updates", please. We can have other Go-related policies here as well.

Reply:
Added.